src/libecalc/core/graph_result.py

```diff
@@ -271,12 +365,22 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
 
         installation_results = self._evaluate_installations(variables_map=self.variables_map)
 
+        regularities: Dict[str, TimeSeriesFloat] = {
```

Comment:
It is very cumbersome to collect, gather the regularities now...was the best (temporary) solution I found without tearing up everything